lib/plugins/inflightRequestThrottle.js

```diff
@@ -65,8 +65,7 @@ function inflightRequestThrottle(opts) {
                 },
                 'maximum inflight requests exceeded, rejecting request'
             );
-            res.send(plugin._err);
-            return next(false);
+            return next(plugin._err);
```

Comment:
I can't comment on the exact line, but I think this file here needs to create the error inline to get a proper stack trace (rather than creating it ahead of time).